lib/socket.js

```diff
@@ -84,7 +85,7 @@ class Transport extends EventEmitter {
 
 const socketFactory = (connect) => {
   const resultConnectFn = (...options) => {
-    const callback = common.extractCallback(options);
+    const callback = common.cbExtract(options);
```

Comment:
Agree, in `metarhia-common` v0.0.21 we have `safeCallback` (renamed `cbExtract`) and `unsafeCallback` (renamed `cbUnsafe`). I placed your code of `extractCallback` into `common.unsafeCallback` because it's optimal implementation. Also I renamed `common.cb` to `once` because `once` will be clear and native for js world.